test/TTBasic/rlexe.xml

```diff
@@ -5,471 +5,471 @@
       <files>accessor.js</files>
       <compile-flags>-TTRecord:!accessorTest -TTSnapInterval:0</compile-flags>
       <baseline>accessorRecord.baseline</baseline>
-      <tags>exclude_dynapogo</tags>
+      <tags>exclude_dynapogo,exclude_jshost,exclude_snap,exclude_serialized</tags>
```

Comment:
+1

---
In reply to: [68102924](https://github.com/Microsoft/ChakraCore/pull/1172#discussion_r68102924) [](ancestors = 68102924)